### PR TITLE
Synchronize two branches: master and live

### PR DESCRIPTION
**Pull request by asm-foysal**
Merge to live branch from master branch.

### CODE REVIEW

**Review by mijin85cho — COMMENTED** [review]
Hello.

I checked this PR, and found some issues that seems to be missed during review for 1522.

Fortunately, they are minor, as like wrong relative path used for the images, etc.

Would you please update the issues, I found?

You can take this PR to your local PC and add an update,
or you can push the fixes to the master, as long as this PR is open, the merged PR to the master will be automatically linked to this PR also.

**Review by asm-foysal — COMMENTED** [review]
@mijin85cho thank you for your review on this PR. We've tried and fixed all the issues you identified and went ahead and gave the rest of the images in PR 1522 another check.

File: docs/application/dotnet/guides/user-interface/nui/videoview.md [hunk 1]
```diff
@@ -16,29 +16,35 @@ The following basic example shows how to create a `VideoView` object:
 
 **Figure: Basic VideoView**
 
-![Basic VideoView](./media/basicVideoView.png)
+![Basic VideoView](../nui-components/media/VideoView.png)
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/dotnet/guides/user-interface/nui/videoview.md:19]: You're right. The image is not in proper place.

If the location of the image is changed, please don't forget user-interface/nui/ui-components.md also, as this page is also using that image.

**Review by mijin85cho — COMMENTED** [review]
Hi @asm-foysal 

Here I reply to you questions. Please check the comments.
Also, you can also check your update in local.

If you open the modified md file with browser with markdown extension (ex. Chrome) you can see how your modification is.


image

If the image/file location is broken, you can check it with your local file also, as it will not on it.

**Comment by mijin85cho** [comment]
@asm-foysal From the request of docs.tizen.org update, 14 days have passed.
Syncing live branch to master is being delayed.

I know you want to manage the issued in this PR,
but it is taking too much time.

Please create a new PR to master to handle the issued,
and merge that PR & this PR.
We need to update the docs.tizen.org.

**Review by asm-foysal — COMMENTED** [review]
@mijin85cho all the issues identified in this PR have been addressed in PR 1603. 
@safir-srbd if everything checks out, please approve this PR for merging.

**Review by mijin85cho — APPROVED** [review]
Thanks for the update.

**Comment by mijin85cho** [comment]
@asm-foysal Thanks for the works for this PR.  I approved!



FYI : We have to sync live branch on 03/31 again, as there would be Tizen Studio release and other related updates.
Please keep in mind, that when we do that, your PR, https://github.com/Samsung/tizen-docs/pull/1600 also should be synced to live.

The person who is leading Tizen Studio release is trying to publish Tizen Studio 03/30 if possible, but we can't sure that schedule for now. I just wanted to let you know that the request could be made on 03/30 or 03/31

**Review by asm-foysal — COMMENTED** [review]
@safir-srbd would you please approve this PR?
@mijin85cho once Safir approves, I'm merging this PR. Thank you for checking this!

**Review by safir-srbd — APPROVED** [review]
@asm-foysal @mijin85cho  I am approving this PR for merging.

**Pull request merged**